cross/fdupes/Makefile

```diff
@@ -0,0 +1,34 @@
+PKG_NAME = fdupes
+
+NAME = fdupes
```

Comment:
As a general comment, please use the same formatting as other packages (wrt order and layout). 
`NAME` isn't defined in spksrc, no need to add it.